plugins/poll/assets/javascripts/widgets/discourse-poll.js.es6

```diff
@@ -333,16 +333,39 @@ createWidget("discourse-poll-container", {
           : "discourse-poll-pie-chart";
       return this.attach(resultsWidget, attrs);
     } else if (options) {
-      return h(
-        "ul",
-        options.map(option => {
-          return this.attach("discourse-poll-option", {
-            option,
-            isMultiple: attrs.isMultiple,
-            vote: attrs.vote
-          });
-        })
+      const contents = [];
+
+      const pollGroups = poll.group_name && poll.group_name.split(",");
```

Comment:
I wonder, why is this called `group_name` when it can hold several groups? Why not call this `groups`?